### PR TITLE
Add stories for team tab components

### PR DESCRIPTION
**Pull request by akalin-keybase**

### CODE REVIEW

File: shared/teams/render.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ type BetaNoteProps = {
   onReadDoc: () => void,
 }
 
-const BetaNote = (props: BetaNoteProps) => (
+export const BetaNote = (props: BetaNoteProps) => (
```
Comment (chrisnojima) [inline shared/teams/render.js]: we usually do an export {Header, BetaNote} at the bottom vs inlining the exports

Comment (akalin-keybase) [reply shared/teams/render.js]: Done

**Comment by akalin-keybase** [comment]
Merging after CI

**Pull request merged**